src/utils/BrowserWorkarounds.ts

```diff
@@ -0,0 +1,14 @@
+import { ChangeEvent } from "react";
+
+type OnChangeHandler<T> = (event: ChangeEvent<HTMLInputElement>) => T;
+
+export function fileOnChangeHandler<T>(handler: OnChangeHandler<T>): OnChangeHandler<T> {
+    return (event) => {
+        const result = handler(event);
+        // Workaround for Chromium Bug
+        // Chrome does not fire onChange events if the same file is selected twice
+        // Only required on Chromium-based browsers (Electron, Chrome, Edge, Opera, Vivaldi, etc)
+        event.target.value = '';
```

Comment:
What's the reason for triggering this at the end of the onChange rather than on the programmatic onClick trigger?

Reply:
Mostly because this way it’s easier to use (just wrap the onChange handler).

Both options work fine, doing it in onClick would reset it just before selecting a new file, doing it in onChange does it just after processing it.

If you’d prefer, I could also modify this PR so the file reset happens in onClick. It’s just a workaround either way, I don’t think there’s going to be a perfect solution.

Reply:
As long as there's no weird browser edge case with programmatic file access after the file has been cleared from the upload input on some weird hardened browser then its fine

Reply:
If we clear on onClick, but the user then cancels the file selection, we’d still get into that same edge case where the value of the input element is empty. So I don’t think we can avoid that situation with either solution.

Reply:
Do we support IE or iOS9 Safari? If so, the onClick option might be preferable, as setting the .value on an input does trigger an onChange event on there.

Reply:
We do not as per https://github.com/vector-im/element-web/#supported-environments
but would that cause an infinite loop? Maybe the code should only set the value if it isn't already empty to prevent that

Reply:
To avoid having to worry about that I’ve just switched to the onClick option now, it’s easier than worrying about recursive event handlers.